camml/megadetector_json_to_csv.py

```diff
@@ -94,8 +90,9 @@ def main():
                         set_type = 'TEST'
 
                     # Get the class name from the image file name
-                    slash = img['file'].find('/')
-                    category = img['file'][0:slash]
+                    slash = img['file'].rfind('/')
+                    prev_slash = img['file'][0:slash - 1].rfind('/')
+                    category = img['file'][prev_slash + 1:slash]
```

Comment:
It occurred to me that we could make these three lines a single line that is may be even a mote easier to read/understand:

`category = img['file'].strip('/').split('/')[-1]`

Do you see any pitfalls with that way of doing it? (there is, of course, the fact that neither approach addresses being on an OS that uses a different slash convention but I think we decided that we're generally not worried about that situation with this project at this stage.)

Reply:
I tried it out and it works with a [-2] index since the structure is "~/class/filename". I don't see any reason why it would cause issues. I'll commit that change.